### DISCUSSION TITLE
Capture indicator blue flags started to appear

### DISCUSSION THREAD

**Discussion by danconde** [azadbolour/eagerwords-community#13] [Q&A] [answered]
![Screen Shot 2023-04-12 at 2 39 35 PM](https://user-images.githubusercontent.com/4314771/231869700-dd9a0cbd-7afb-4da5-b5e5-716decbec436.png)

Blue flags to indicate capture risk have started to appear, although they rarely did before. If it's behaving properly now, it's fine, but in case this is happening via unintentional code change, this is an FYI

**Comment by azadbolour** [comment, marked as answer]
There have not been any recent changes to the algorithms or the user
interface.

Just for completeness: the blue flags should appear in a position when and
only when you are forming a word, and that position is going to be
captured. If you see this condition violated, please send a report
including a screenshot and the entire browser console for that game so I
can pinpoint and fix the issue.

Thanks.

Azad
